src/pages/index.tsx

```diff
@@ -42,94 +41,41 @@ const IndexPage = () => (
     <Highlights
       highlight={[
         {
-          visual: 'hydra',
-          title: 'OAuth 2.0 and OpenID Connect',
-          href: '/hydra',
+          visual: 'opensource',
+          title: 'Powered by Open Source',
+          href: 'https://github.com/ory',
+          openInNewWindow: true,
           content:
-            'Keep threats out and easily verify who is a customer, and customize sign in flows that fit you and your customers.'
+            'You can rely on open source software that is built together in a world-wide community of developers. Peer reviewed, vetted by expert developers - tried and tested.'
```

Comment:
I like the imperative more, also the other texts are imperative
```suggestion
            'Rely on open source software that is built together in a world-wide community of developers. Peer reviewed, vetted by expert developers - tried and tested.'
```